src/carousel/carousel.component.ts

```diff
@@ -164,6 +165,65 @@ export class CarouselComponent implements OnDestroy {
     this.activeSlide = this.findNextSlideIndex(Direction.PREV, force);
   }
 
+  /**
+   * Swith slides by enter, space and arrows keys
```

Comment:
all these methods shouldn't be present in the API doc section, so remove these comments (or mark them as `@internal`)